app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      def initialize(work, attributes)
+        @work = work
```

Comment:
Done. I was aware of the "_" syntax, but didn't know if it was better to remove it. Thanks ;-)